### ISSUE TITLE
Dockerfile error

### ISSUE THREAD

**Issue by bkj** [facebookresearch/wav2letter#147]
The docker build runs for a while, then hits
```
[ 69%] Linking CXX executable DctTest
[ 69%] Building CXX object src/tests/CMakeFiles/SoundTest.dir/__/runtime/Distributed.cpp.o
[ 70%] Linking CXX executable TriFilterbankTest
/usr/local/lib/libsndfile.a(flac.c.o): In function `sf_flac_error_callback':
flac.c:(.text+0x907): undefined reference to `FLAC__StreamDecoderErrorStatusString'
/usr/local/lib/libsndfile.a(flac.c.o): In function `flac_seek':
flac.c:(.text+0x99d): undefined reference to `FLAC__stream_decoder_seek_absolute'
/usr/local/lib/libsndfile.a(flac.c.o): In function `flac_close':
```

Any ideas on how to fix?

**Comment by jacobkahn** [comment]
@bkj — we're aware of this issue and should have a fix up today. The temporary workaround is to install libsndfile with autogen/autoconf (as listed at the bottom [here](https://github.com/erikd/libsndfile#hacking) instead of with Git/their CMake build.

**Comment by bkj** [comment]
Great -- thanks!  Will test once you push the fix.

**Comment by jacobkahn** [comment]
@bkj — the fix is in 0cd8b0b4d893c0b1563c452e2c432f010b28db36. Feel free to reopen if things still aren't working.

**Comment by tlikhomanenko** [comment]
@bkj I have updated the `Dockerfile-CUDA`, so you can use it. Also the docker images are available here  https://hub.docker.com/r/wav2letter/wav2letter/tags (for both CUDA and CPU backends now).
